### PR TITLE
Initial implementation of ActiveMQ component

### PR DESCRIPTION
**Pull request by troycrews**
The MessageQueueComponentTest JUnit test class confirms that the Camel ActiveMQ component is operational, wired in to the project setup, and able to create and manipulate queues.

### CODE REVIEW

**Comment by eedrummer** [comment]
I'm not sure that I understand what is going on with the implementation. It looks like it is writing messages into a database. I would think we would want it to write `SDPMessage`s into ActiveMQ.

**Comment by troycrews** [comment]
I had some cruft in there from a previous add step. I cleaned out the initial consumer and producer files, as they're not needed in the route context.

**Comment by eedrummer** [comment]
That makes sense. I don't think the test really covers what we would want to check. I expected to see something similar to https://github.com/CDCgov/SDP-CBR/blob/master/database_queue/src/test/java/gov/cdc/sdp/cbr/queue/DatabaseQueueComponentTest.java

In that case, there is a running instance of Postgres and a whole Camel context is set up here: https://github.com/CDCgov/SDP-CBR/blob/master/database_queue/src/test/resources/DatabaseQueueComponentTest-context.xml

I would expect us to have a test where we put a message into running instance of ActiveMQ and then show that it gets delivered to a mock endpoint. We should also have a test where we read in a message and then go into ActiveMQ directly to ensure the message is there.

The test suite should fail if ActiveMQ is not running locally, as it does with Postgres.

**Comment by troycrews** [comment]
We now have the ability to post an SDP message in a route to queue #1, transform and pickup/deliver to queue #2, and assert the resulting endpoint message count.

File: database_queue/src/test/java/gov/cdc/sdp/cbr/queue/MessageQueueRouteTest.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        map.put("arguments", "arge");
+        map.put("fromPartyId", "testQueueProducer");
+        map.put("messageRecipient", "recipient");
+        
+        resultEndpoint.expectedBodiesReceived(map);
+        resultEndpoint.message(0).header("cheese").isEqualTo(123);
+
+        sendExchange(map);
+
+        //resultEndpoint.assertIsSatisfied();
```
Comment (eedrummer) [inline database_queue/src/test/java/gov/cdc/sdp/cbr/queue/MessageQueueRouteTest.java]: Why is this commented out?

**Comment by troycrews** [comment]
The following line (now line 92) does the assertion the context-aware way, which is preferred. Deleted commented line.

**Pull request merged**